api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -1,25 +1,27 @@
 #!/bin/bash
 
-# This generates big query count databases cdr and public that get put in cloudsql for workbench and data browser
+# This generates big query count databases cdr that get put in cloudsql for workbench
 
 set -xeuo pipefail
 IFS=$'\n\t'
 
-
 # get options
 # --project=all-of-us-workbench-test *required
 
 # --cdr=cdr_version ... *optional
-USAGE="./generate-clousql-cdr/make-bq-data.sh --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT>"
+USAGE="./generate-clousql-cdr/make-bq-data.sh --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT> --public-project <PROJECT>"
```

Comment:
Perhaps this should just be a more generic --output-project --output-dataset; then let the caller specify "cdr20181025" vs "public20181025". I think other than the dataset name this script doesn't really care where it's writing to.

Reply:
Makes sense. Changed it.